packages/svelte-query/package.json

```diff
@@ -34,7 +34,7 @@
     "tslib": "^2.4.1",
     "typescript": "^4.7.4",
     "vite": "^4.0.0",
-    "vitest": "^0.26.3"
+    "vitest": "^0.27.1"
```

Comment:
the `"@vitest/coverage-istanbul"` dependency matched the version of `vitest`. not sure if that's required but maybe we should update it as well?

Reply:
good catch
[7d3ad6a](https://github.com/TanStack/query/pull/4825/commits/7d3ad6a5b1ed99cdc226a763a180ba30074f07cd)